app/views/bars/edit.html.haml

```diff
@@ -1,7 +1,8 @@
-%h1{:align => "center"} Edit #{@bar.name}
-
-= render 'form'
-
-[
-= link_to "I've changed my mind. This bar looks cool.", bars_path
-]
+.container
+  %h1{:align => "center"} Edit #{@bar.name}
```

Comment:
{ align: "center" } instead of {:align => "center"} to keep same style everywhere maybe? ( => are used everywhere in here, not only this particular line)

Reply:
Consider avoiding inline styling. Better way is to create scss class for that. What's more, it might be added with bootstrap so please check docs for that.